<ui version="4.0" >
 <class>EditFile</class>
 <widget class="QWidget" name="EditFile" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>901</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Edit File</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" colspan="2" >
    <widget class="QTextBrowser" name="QTBEditFile" >
     <property name="minimumSize" >
      <size>
       <width>530</width>
       <height>800</height>
      </size>
     </property>
     <property name="font" >
      <font>
       <family>Courier New</family>
      </font>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>14</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="GroupButtons" >
     <property name="title" >
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QPushButton" name="buttonQuit" >
        <property name="text" >
         <string>&amp;Quit</string>
        </property>
        <property name="autoDefault" >
         <bool>true</bool>
        </property>
        <property name="default" >
         <bool>true</bool>
        </property>
        <property name="flat" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QPushButton" name="buttonPrint" >
        <property name="text" >
         <string>&amp;Print</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="buttonHelp" >
        <property name="text" >
         <string>&amp;Help</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>249</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
